go/teams/transactions.go

```diff
@@ -642,6 +643,22 @@ func (tx *AddMemberTx) Post(ctx context.Context) (err error) {
 		if !found {
 			return fmt.Errorf("AddMemberTx.Post got a PerTeamKey but couldn't find a link with None to attach it")
 		}
+	} else { // If we didn't rotate the PTK, let's rebox the existing PTK for any of the new members
+		ekLib := g.GetEKLib()
+		if ekLib != nil {
+			// Only box up for the latest members
+			memSet.removeExistingMembers(ctx, team)
+			i := 0
+			uids := make([]keybase1.UID, len(memSet.recipients))
+			for uv := range memSet.recipients {
+				uids[i] = uv.Uid
+				i++
+			}
+			teamEKBoxes, err = ekLib.BoxLatestTeamEK(ctx, team.ID, uids)
+			if err != nil {
+				return err
```

Comment:
Double checking that you want this to be fatal.

Reply:
@oconnor663 what are your thoughts?

Reply:
added a check for no boxes, otherwise let's propagate